astropy/coordinates/representation/cylindrical.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import annotations
```

Comment:
I think all files should include the license line at the top, so make sure you add these to new files here. I wonder if we can add this check to pre-commit?

Reply:
We could make a pre-commit hook that can autocorrect this.

Reply:
https://pypi.org/project/pre-commit-license-headers/

Reply:
Closing the because we have #13786

Reply:
So, why is the license file still not added automatically then?